packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -199,7 +208,9 @@ export class FabricTestLedgerV1 implements ITestLedger {
     const fnTag = `${this.className}#getConnectionProfileOrg1()`;
     const cInfo = await this.getContainerInfo();
     const container = this.getContainer();
-    const ccpJsonPath = "/fabric-samples/first-network/connection-org1.json";
+    const ccpJsonPath = this.getFabricVersion().valueOf().includes("1.4.")
```

Comment:
There shouldn't be a need to call the `valueOf()` method, the `string` type already has the includes method on it IIRC (happy to be wrong on this one I did not actually test it just typing from memory)

Reply:
Incorporated the changes. Please have a re-look.